vivisect/analysis/elf/elfplt.py

```diff
@@ -170,19 +158,20 @@ def getPLTs(vw):
     for fname in vw.getFiles():
         fdyns = vw.getFileMeta(fname, 'ELF_DYNAMICS')
         if fdyns is not None:
-            FGOT = fdyns.get('DT_JMPREL')
-            FGOTSZ = fdyns.get('DT_PLTRELSZ')
-            if vw.getFileMeta(fname, 'addbase'):
-                imgbase = vw.getFileMeta(fname, 'imagebase')
-                logger.debug('Adding Imagebase: 0x%x', imgbase)
-                FGOT += imgbase
+            FPLT = fdyns.get('DT_JMPREL')
+            FPLTSZ = fdyns.get('DT_PLTRELSZ')
+
+            # if we don't have FPLT or FPLTSZ, skip this
+            if None in (FPLT, FPLTSZ):
+                continue
 
             newish = True
             for pltva, pltsize in plts:
-                if FGOT == pltva:
+                if FPLT == pltva:
                     newish = False
-            if newish and FGOT and FGOTSZ:
-                plts.append((FGOT, FGOTSZ))
+
+            if newish:
+                plts.append((FPLT, FPLTSZ))
```

Comment:
while changing from 
```
            if vw.getFileMeta(fname, 'addbase'):
                imgbase = vw.getFileMeta(fname, 'imagebase')
```
to 
```
            imgbase += imageoff
```
i just cleaned this up a little and renamed some things that are hopefully more clear.